src/App.tsx

```diff
@@ -14,11 +16,141 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+interface State {
+  goods: string[];
+  showList: boolean;
+  value: string;
+}
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    goods: [...goodsFromServer],
+    showList: false,
+    value: `${goodsFromServer.length}`,
```

Comment:
you made this value a string and then in the method on line 80 you convert it back to number. seems a bit counterproductive